google-apis-core/spec/google/apis/options_spec.rb

```diff
@@ -37,4 +38,15 @@
   it 'should allow nil in merge' do
     expect(options.merge(nil)).to be_an_instance_of(Google::Apis::RequestOptions)
   end
+
+  it 'sets default values' do
+    expect(defaults.retries).to eq(5)
```

Comment:
This value should be [0](https://github.com/googleapis/google-api-ruby-client/blob/7f61f108cbd32b0c5eaac23768d42960e59998a0/google-apis-core/lib/google/apis/options.rb#L111)

Reply:
I thought so too, but `spec_helper.rb` overrides this. 😄